fsharp-backend/src/LibBinarySerialization/SerializedTypesToProgramTypes.fs

```diff
@@ -65,8 +78,27 @@ module Expr =
       PT.EFieldAccess(id, toPT obj, fieldname)
     | ST.EFnCall (id, name, args, ster) ->
       PT.EFnCall(id, FQFnName.toPT name, List.map toPT args, SendToRail.toPT ster)
-    | ST.EBinOp (id, name, arg1, arg2, ster) ->
-      PT.EBinOp(id, FQFnName.toPT name, toPT arg1, toPT arg2, SendToRail.toPT ster)
+    | ST.EBinOp (id, ST.FQFnName.Stdlib fn, arg1, arg2, ster) ->
+      assertIn
+        "serialized binop should have blank/Date module"
+        [ ""; "Date" ]
```

Comment:
should we ad a CLEANUP to this specific line? (I think we're looking to make Date fns not allow this form?)

Reply:
Added.